ci-operator/step-registry/redhat-chaos/node-disruptions/worker-outage/redhat-chaos-node-disruptions-worker-outage-ref.yaml

```diff
@@ -12,6 +12,9 @@ ref:
   - namespace: test-credentials
     name: ocp-qe-perfscale-es
     mount_path: /secret/es
+  - namespace: test-credentials
+    name: devqe-secrets
```

Comment:
the name here for the profile we are using should be https://github.com/openshift/ci-tools/blob/e7e8a8cb02b2b39797fdfacfe2f1a25e5ef0cb27/pkg/load/load_test.go#L311 I think

Reply:
I don't think you actually will need this. Hopefully setting what was the vsphere_center to vsphere_ip will be the only change you need